example/randomCells2D/src/randomCells2D.cpp

```diff
@@ -202,16 +202,17 @@ auto example(TAccTag const&) -> int
 
     auto pitchBufAccRandV = alpaka::getPitchesInBytes(bufAccRandV)[0];
 
-    auto const& bundeledKernelInitRandom
+    auto const& kernelBundleInitRandom
         = alpaka::KernelBundle(initRandomKernel, extent, ptrBufAccRandS, pitchBufAccRandS);
     // Let alpaka calculate good block and grid sizes given our full problem extent
     auto const workDivInitRandom
-        = alpaka::getValidWorkDivForKernel<Acc>(devAcc, bundeledKernelInitRandom, extent, Vec(perThreadY, perThreadX));
+        = alpaka::getValidWorkDivForKernel<Acc>(devAcc, kernelBundleInitRandom, extent, Vec(perThreadY, perThreadX));
 
     alpaka::exec<Acc>(queue, workDivInitRandom, initRandomKernel, extent, ptrBufAccRandS, pitchBufAccRandS);
     alpaka::wait(queue);
 
-    alpaka::exec<Acc>(queue, workDivInitRandom, initRandomKernel, extent, ptrBufAccRandV, pitchBufAccRandV);
+    // execute the same kernel with different pointers
+    alpaka::exec<Acc>(queue, workDivInitRandom, kernelBundleInitRandom);
```

Comment:
Isn't the previous call the one that should be replace with the `kernelBundle` ?
This one uses different arguments.

Reply:
well catched, thats a bug I introduced.